### PR TITLE
Add options for board creation

### PR DESCRIPTION
**Pull request by joefatora**
Added some params for organization ID, Default Lists, Permission Level, Self Join. Includes a sanity check since setting perm level to 'org' requires an org ID be passed too

### CODE REVIEW

File: src/translator.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ Translator.prototype.reloadTranslations = function(type, onComplete) {
           "] (this may take a while)"
       );
       _.each(memberships, function(m) {
-        getMembershipsThrottled(m.idMember, function(err, user) {
+        getMemberships(m.idMember, function(err, user) {
```
Comment (mheap) [inline src/translator.js]: If I remember correctly, the throttling is required for organisations with lots of members, or we hit a rate limit. Was it causing issues?

Comment (joefatora) [inline src/translator.js]: no I don't think so, I was just testing something and forgot that this PR hadn't been closed. I'll revert the commit.

**Comment by mheap** [comment]
Thanks for the contribution 👍

**Comment by joefatora** [comment]
Thanks for making an awesome tool 😄

**Pull request merged**